projects/storefrontlib/src/shared/components/media/media.service.ts

```diff
@@ -52,7 +52,8 @@ export class MediaService {
   getMedia(
     mediaContainer: MediaContainer | Image,
     format?: string,
-    alt?: string
+    alt?: string,
+    role?: string
   ): Media {
```

Comment:
I see you were able to target the `<img>` element, that's great! However, as `role="presentation"` will override the `alt` tag, only one solution should be used. 
The preferred solution is to have an empty `alt=""`, but if a null value is not possible, then please remove the `alt` tag and use `role="presentation"` only.

<img width="614" alt="Screen Shot 2021-06-28 at 4 14 30 PM" src="https://user-images.githubusercontent.com/15113219/123700135-10fc5680-d82e-11eb-95b4-a66234b5f102.png">

Reply:
See **Example 1** in this link for reference on using either `alt=""` or  `role="presentation"` https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
empty `alt` attribute gets stripped during rendering so removed it and kept `role="presentation"`